apps/i18n/common/en_us.json

```diff
@@ -41,8 +41,7 @@
   "addStudentsMultipleInstructions": "Type or paste the names of your students, one per line.",
   "addStudentsManageMyOwn": "Create and manage my own list of students",
   "addStudentsSyncThirdParty": "Sync my list of students from an existing classroom section in a third party tool",
-  "addStudentsToSectionInstructions": "Choose how you want to add your students:",
-  "addStudentsToSectionInstructionsUpdated": "How do you want your students to sign in?",
+  "addStudentsToSectionInstructions": "How do you want your students to sign in?",
```

Comment:
Are we worried about losing the pre-existing translations by switching around the string keys?

Reply:
Oh yes ur right thank you! I'll update that